library/core/src/main/java/com/google/android/exoplayer2/video/MediaCodecVideoRenderer.java

```diff
@@ -378,7 +378,7 @@ private void setSurface(Surface surface) throws ExoPlaybackException {
       // The surface is set and unchanged. If we know the video size and/or have already rendered to
       // the surface, report these again immediately.
       maybeRenotifyVideoSizeChanged();
-      maybeRenotifyRenderedFirstFrame();
+      maybeRenotifyRenderedFirstFrame(0);
```

Comment:
You should probably pass the timestamp of the last frame that was rendered here, since that's the frame that's currently displayed on the surface.